src/main/java/Duke.java

```diff
@@ -1,10 +1,165 @@
+        listCount++;
+        System.out.print("Now you have " + listCount + " task");
+        if(listCount > 1) {
+            System.out.print("s");
+        }
+        System.out.println(" in the list.");
+        return listCount;
+    }
+    public static void completeTask (Todo[] recordedList, int index) {
+        recordedList[index].setAsDone();
+        System.out.println("Nice! I've marked this task as done:");
+        printTask(recordedList, (index));
+
+    }
+    public static void bidGoodbye () {
+        printHorizontal();
+        System.out.println("Bye. Hope to see you again soon!");
+        printHorizontal();
+    }
+
+    public static void main(String[] args) {
+        Todo[] recordedList = new Todo[100];
+        int listCount = 0;
+
+        printHeader();
+
+        Scanner input = new Scanner(System.in);
+        String command = input.nextLine();
+        String keyword = extractKeyword(command.trim());
+        String details;
+        if(findCommandLength(command) > 1) {
+            details = extractDetails(command.trim());
+        }
+        else {
+            details = " ";
+        }
+
+        while (!isBye(keyword)) {
+            switch(keyword) {
+            case "todo":
+                printHorizontal();
+                listCount = addTodoToList(recordedList, details, listCount);
+                printHorizontal();
+                break;
+            case "deadline":
+                printHorizontal();
+                listCount = addDeadlineToList(recordedList, details, listCount);
+                printHorizontal();
+                break;
+            case "event":
+                printHorizontal();
+                listCount = addEventToList(recordedList, details, listCount);
+                printHorizontal();
+                break;
+            case "list":
+                printHorizontal();
+                printList(recordedList, listCount);
+                printHorizontal();
+                break;
+            case "done":
+                int index = Integer.parseInt(details);
+                if (index > listCount) {
+                    printHorizontal();
+                    System.out.println("Sorry, there is no such item.");
+                    printHorizontal();
+                }
+                else {
+                    printHorizontal();
+                    completeTask(recordedList, (index -1));
+                    printHorizontal();
+                }
+                break;
+            default:
+                printHorizontal();
+                System.out.println("Sorry, this command does not exist!");
+                printHorizontal();
+            }
+            command = input.nextLine();
+            keyword = extractKeyword(command.trim());
+            if(findCommandLength(command) > 1) {
+                details = extractDetails(command.trim());
+            }
+            else {
+                details = " ";
+            }
+        }

```

Comment:
I like how the main method is easy to read, however, perhaps you could consider separating the main method to several other methods so that it is shorter?